ivy.xml

```diff
@@ -90,6 +90,69 @@
     <dependency org="org.owasp" name="dependency-check-ant"
                 rev="${dependency-check-ant.version}" conf="owasp->default"/>
 
+    <dependency org="commons-io" name="commons-io"
+                rev="${commons-io.version}" conf="test->default"/>
+
+      <dependency org="org.apache.kerby" name="kerby-config" rev="${kerby.version}" conf="test->default">
+          <exclude org="org.slf4j" module="slf4j-api"/>
```

Comment:
Good work! Thanks @afine for your efforts. 
Apart from @phunt comment, I've below observation.

Should we need to exclude 'slf4j-api' and 'slf4j-log4j12' from all the kerb dependency, which is missing in branch-3.5. If needed then could you please raise a minor imprv task to correct in branch-3.5 and push it in together, that would syncup the code between branches.

Reply:
I reduced the number of excludes to the minimum that I think is necessary. Not sure why the ivy "stict" resolution change is not in 3.5. I'll investigate

Reply:
And create a jira to make the branches the same if necessary